selfdrive/car/nissan/carcontroller.py

```diff
@@ -76,7 +76,7 @@ def update(self, enabled, CS, frame, actuators, cruise_cancel, hud_alert,
         can_sends.append(nissancan.create_cancel_msg(self.packer, CS.cancel_msg, cruise_cancel))
 
     can_sends.append(nissancan.create_steering_control(
-        self.packer, self.car_fingerprint, apply_angle, frame, acc_active, self.lkas_max_torque))
+        self.packer, self.car_fingerprint, apply_angle, frame, enabled, self.lkas_max_torque))
```

Comment:
@avolmensky do you know any reason why we set the apply bit with the cruise state? Can we just set it with the steering state (`CC.latActive`)? The only difference would be at a standstill, the bit would now fall, and for any steering faults (which we do not log right now, so no difference there).